src/Servers/IIS/IIS/test/Common.FunctionalTests/LogFileTests.cs

```diff
@@ -78,8 +78,8 @@ public async Task InvalidFilePathForLogs_ServerStillRuns(TestVariant variant)
             StopServer();
             if (variant.HostingModel == HostingModel.InProcess)
             {
-                EventLogHelpers.VerifyEventLogEvent(deploymentResult, "Could not start stdout redirection in (.*)aspnetcorev2.dll. Exception message: HRESULT 0x80070003");
-                EventLogHelpers.VerifyEventLogEvent(deploymentResult, "Could not stop stdout redirection in (.*)aspnetcorev2.dll. Exception message: HRESULT 0x80070002");
+                // Error is getting logged twice, from shim and handler
+                EventLogHelpers.VerifyEventLogEvent(deploymentResult, EventLogHelpers.CouldNotStartStdoutFileRedirection("Q:\\std", deploymentResult), allowMultiple: true);
```

Comment:
Do our current tests handle the following:
- Having main hanging and reading output from ForwardingRedirectionOutput
- Getting output from the new error callback

Reply:
It's hard to do both. I don't know of a scenario where hostfxr would error out while still starting clr.